numbers/search-available.php

```diff
@@ -9,8 +9,17 @@
 $client = new \Vonage\Client($basic);
 
 /** @var IterableAPICollection $response */
+/** Note: be careful when specifying "NUMBER_SEARCH_CRITERIA" environment variable: 
+  * surround it with quotes if it has a leading zero, otherwise the (string) conversion
+  * will assume it is an Octal number and convert it.  e.g., (string)0123 results in "83".
+  * NUMBER_SEARCH_PATTERN will be one of 0 (search for numbers that start with pattern);
```

Comment:
I'm good with the warning above, but an we remove the options listed here? The allowed options are all listed on https://developer.nexmo.com/numbers/code-snippets/search-available/php, and it's expected that these snippets are being used with the documentation versus stand-alone.